tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -654,16 +654,16 @@
     <comment>Tooltip for the system memory usage box in the bar</comment>
   </data>
   <data name="PIDisplayName" xml:space="preserve">
-    <value>Project Ironsides</value>
-    <comment>{Locked}The name of the Project Ironsides application</comment>
+    <value>Dev Insights</value>
+    <comment>{Locked}The name of the Dev Insights application</comment>
   </data>
   <data name="PIAppDescription" xml:space="preserve">
-    <value>Dev Home Project Ironsides</value>
-    <comment>{Locked}The description of the Project Ironsides application</comment>
+    <value>Dev Insights</value>
+    <comment>{Locked}The description of the Dev Insights application</comment>
   </data>
   <data name="PIStartupTaskDisplayName" xml:space="preserve">
-    <value>Hotkey for Dev Home Project Ironsides</value>
-    <comment>{Locked="Ironsides"} The description of the startup task for Dev Home Project Ironsides. This shows up in task manager/Windows Settings &gt; Startup Apps.</comment>
+    <value>Hotkey for Dev Insights</value>
+    <comment>The description of the startup task for Dev Insights. This shows up in task manager/Windows Settings &gt; Startup Apps.</comment>

```

Comment:
Sorry to be annoying but I think you need to add a lock back in here :) Looking down the changes, it seems when it's the full string it's locked but not when it's part of a string.

Reply:
I'll reach out to branding to make sure we should be locking the entire name, I could see it being beneficial if "Dev Insights" was translated.